front/app/components/PostShowComponents/Comments/ChildCommentForm.tsx

```diff
@@ -184,7 +184,7 @@ const ChildCommentForm = ({
     setCanSubmit(focused && !hasEmptyError);
   };
 
-  const onFocus = () => {
+  const onClick = () => {
```

Comment:
On focus might actually be better in this case as it will handle tab navigation too 🤔

Reply:
Could you elaborate? Not sure what you mean exactly.

Reply:
I mean that we are tracking `focusChildCommentEditor`. If we use onClick then we are only capturing it for clicks and not necessarily navigation using tabs